app/scripts/modules/titus/src/pipeline/stages/runJob/TitusRunJobStageConfig.tsx

```diff
@@ -77,6 +87,16 @@ export class TitusRunJobStageConfig extends React.Component<IStageConfigProps, I
     const clusterDefaults: IClusterDefaults = {
       application: application.name,
       containerAttributes: {},
+      constraints: {
+        soft: {},
+        hard: {},
+      },
+      efs: {
+        efsId: string,
+        mountPoint: string,
+        mountPerm: string,
+        efsRelativeMountPoint: string,
```

Comment:
looks like these `string` values are written like they're in a typescript interface, but this is an object. should they be `''` instead?

Reply:
Makes sense , moved it to `IClusterDefaults`, let me know if this is better.

Reply:
`IClusterDefaults` interface looks good, just be aware that if you want this `clusterDefaults` object to have initial values for those fields you'll still need to initialize them to empty stings (i.e. `''`) here too. If you don't care about those fields being present in the object when they're not explicitly added by users, then that's fine!

Reply:
Yes we don't want to have those by default, so those initial values are not required in this case.

Reply:
👍 👍